tensorflow_serving/batching/batching_session.cc

```diff
@@ -29,6 +31,7 @@ limitations under the License.
 #include "tensorflow_serving/servables/tensorflow/serving_session.h"
 #include "tensorflow_serving/util/cleanup.h"
 #include "tensorflow_serving/util/hash.h"
+#include "third_party/eigen3/unsupported/Eigen/CXX11/Tensor"
```

Comment:
Hmmm. I'm not aware of any other place in the tf-serving codebase where we import eigen directly. Are you sure what you want to do can't be accomplished via the tensorflow layer? (If needed we could potentially add a utility in tensorflow.)